MapboxNavigation/MGLStyle.swift

```diff
@@ -1,24 +1,26 @@
 import Foundation
 import Mapbox
 
+
 extension MGLStyle {
     // Returns the URL to the current version of the Mapbox Navigation Guidance Day style.
     @objc
-    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: "mapbox://styles/mapbox/navigation-guidance-day-v2")! } }
+    public class var navigationGuidanceDayStyleURL: URL { get { return URL(string: (MGLAccountManager.hasChinaBaseURL ? MGLAccountManager.mapboxChinaDayStyleURL : "mapbox://styles/mapbox/navigation-guidance-day-v2"))! } }
     
     // Returns the URL to the current version of the Mapbox Navigation Guidance Night style.
     @objc
-    public class var navigationGuidanceNightStyleURL: URL { get { return URL(string: "mapbox://styles/mapbox/navigation-guidance-night-v2")! } }
+    public class var navigationGuidanceNightStyleURL: URL { get { return URL(string: (MGLAccountManager.hasChinaBaseURL ? MGLAccountManager.mapboxChinaNightStyleURL : "mapbox://styles/mapbox/navigation-guidance-night-v2"))! } }
     
     @objc
     // Returns the URL to the given version of the navigation guidance style. Available version are 1, 2, and 3.
```

Comment:
Add a note that the version is ignored when the API base URL is api.mapbox.cn. (×2)

Also, while you’re here, would you mind turning all four of these comments into proper documentation comments using the `/**` syntax? Thanks!

Reply:
Sure.